$MirOS: ports/www/php/patches/patch-ext_iconv_php_iconv_h,v 1.1 2006/02/21 16:25:21 tg Exp $
--- ext/iconv/php_iconv.h.orig	Mon Dec  6 20:38:33 2004
+++ ext/iconv/php_iconv.h	Tue Feb 21 16:55:13 2006
@@ -36,6 +36,7 @@
 #include "ext/iconv/php_have_iconv.h"
 #include "ext/iconv/php_have_libiconv.h"
 #include "ext/iconv/php_have_glibc_iconv.h"
+#include "ext/iconv/php_have_citrus_iconv.h"
 #include "ext/iconv/php_have_bsd_iconv.h"
 #include "ext/iconv/php_iconv_supports_errno.h"
 #include "ext/iconv/php_php_iconv_impl.h"
